### ISSUE TITLE
how to stop logging while running kasm containers ?

### ISSUE THREAD

**Issue by jakbin** [kasmtech/workspaces-images#68]
when i run kasm containers , its continuously print logging data on terminal . How i can stop this ?

**Comment by j-travis** [comment]
There is a standard flag for docker run `--detach` aka `-d` . You can pass this so the container runs in the background and you dont have to see the logs.

https://docs.docker.com/engine/reference/commandline/run/#options

**Comment by jakbin** [comment]
no, it's not working.If i put like this `docker run --rm -i  --shm-size=512m -p 6901:6901 -e VNC_PW=password1 kasmdemo --detach` it's not stopping. And if i put like this `docker run --rm -i --detach  --shm-size=512m -p 6901:6901 -e VNC_PW=password1 kasmdemo` then vnc is not running. 

@j-travis
